### ISSUE TITLE
Board template

### ISSUE THREAD

**Issue by jordanwong** [BouncingSquaresNew/HexaboxesRepo#1]
Something like this? :)
![hexagonal_2](https://cloud.githubusercontent.com/assets/10156330/5436863/f95c0866-8462-11e4-9889-1ba5faf8df8f.png)

**Comment by ChristianLoizou** [comment]
Yeah that looks good. Just need to set it as the background for the app and
then build the lines around the coordinates where the dots are.
